cmd/worker/workercmd/cmd.go

```diff
@@ -2,8 +2,9 @@ package workercmd
 
 import (
 	"context"
+	"fmt"
 
-	"github.com/artefactual-sdps/temporal-activities/bagit"
+	"github.com/artefactual-sdps/temporal-activities/bagcreate"
```

Comment:
Updating temporal-activities after the re-org. I'll do it in other projects too ;)